topics/single-cell/tutorials/GO-enrichment/tutorial.md

```diff
@@ -134,7 +134,7 @@ To perform GO enrichment analysis on each cell cluster individually, we need to
 > 1. {% tool [Split file](toolshed.g2.bx.psu.edu/repos/bgruening/split_file_on_column/tp_split_on_column/0.4) %} with the following parameters:
 >    - {% icon param-file %} *"File to select"*: `Markers_cluster` (Input dataset)
 >    - *"on column"*: `c1`
->    - *"Include the header in all splitted files?"*: `Yes`
+>    - *"Include the header in all split files?"*: `Yes`
```

Comment:
```suggestion
>    - *"Include the header in all splitted files?"*: `Yes`
```

Reply:
Unfortunately, the tool itself says 'splitted', so we have to make sure the tutorial text matches the tool text in the Graphical User Interface / Galaxy interface, otherwise people can get confused.